cmd/utils.go

```diff
@@ -93,6 +93,22 @@ func path2BucketObject(s string) (bucket, prefix string) {
 	return path2BucketObjectWithBasePath("", s)
 }
 
+func getParityBlocks(drive int) int {
```

Comment:
call this like this?
getDefaultParityBlocks
getDefaultParityBlocks

because otherwise it's dependent on EC config.